src/dev/build/build_distributables.ts

```diff
@@ -107,7 +107,8 @@ export async function buildDistributables(log: ToolingLog, options: BuildOptions
   }
   if (options.createDockerPackage) {
     // control w/ --docker or --skip-docker-ubi or --skip-os-packages
-    await run(Tasks.CreateDockerPackage);
+    await run(Tasks.CreateDockerX64Package);
```

Comment:
I think we should move this outside of the `--docker` flag. Creating the contexts are fast, and that flag was to avoid the lengthy Docker build.

Reply:
Pushed with fcccf98. 

Part of the changes here removed the UBI only flag:
It's sitting at a different level of abstraction than the other flags and was getting unwieldy, moreso with the ironbank context.  We don't have the same granularity for archives and OS packages and I don't think it'll be an especially common build scenario.

Wondering what your thoughts are - we could add it back, or hoist specific builds to the task level.